addons/l10n_br/data/res.city.csv

```diff
@@ -1,5571 +1,5571 @@
-city_br_5026,base.br,base.state_br_rs,Sentinela do Sul
-city_br_5027,base.br,base.state_br_ba,Sento Sé
-city_br_5028,base.br,base.state_br_rs,Serafina Corrêa
-city_br_5029,base.br,base.state_br_mg,Sericita
-city_br_5030,base.br,base.state_br_ro,Seringueiras
-city_br_5031,base.br,base.state_br_rs,Sério
-city_br_5032,base.br,base.state_br_mg,Seritinga
-city_br_5033,base.br,base.state_br_rj,Seropédica
-city_br_5034,base.br,base.state_br_sc,Serra Alta
-city_br_5035,base.br,base.state_br_sp,Serra Azul
-city_br_5036,base.br,base.state_br_mg,Serra Azul de Minas
-city_br_5037,base.br,base.state_br_pb,Serra Branca
-city_br_5038,base.br,base.state_br_rn,Serra Caiada
-city_br_5039,base.br,base.state_br_pb,Serra da Raiz
-city_br_5040,base.br,base.state_br_mg,Serra da Saudade
-city_br_5041,base.br,base.state_br_rn,Serra de São Bento
-city_br_5042,base.br,base.state_br_rn,Serra do Mel
-city_br_5043,base.br,base.state_br_ap,Serra do Navio
-city_br_5044,base.br,base.state_br_ba,Serra do Ramalho
-city_br_5045,base.br,base.state_br_mg,Serra do Salitre
-city_br_5046,base.br,base.state_br_mg,Serra dos Aimorés
-city_br_5047,base.br,base.state_br_ba,Serra Dourada
-city_br_5048,base.br,base.state_br_pb,Serra Grande
-city_br_5049,base.br,base.state_br_sp,Serra Negra
-city_br_5050,base.br,base.state_br_rn,Serra Negra do Norte
-city_br_5051,base.br,base.state_br_mt,Serra Nova Dourada
-city_br_5052,base.br,base.state_br_ba,Serra Preta
-city_br_5053,base.br,base.state_br_pb,Serra Redonda
-city_br_5054,base.br,base.state_br_pe,Serra Talhada
-city_br_5055,base.br,base.state_br_sp,Serrana
-city_br_5056,base.br,base.state_br_mg,Serrania
-city_br_5057,base.br,base.state_br_ma,Serrano do Maranhão
-city_br_5058,base.br,base.state_br_go,Serranópolis
-city_br_5059,base.br,base.state_br_mg,Serranópolis de Minas
-city_br_5060,base.br,base.state_br_pr,Serranópolis do Iguaçu
-city_br_5061,base.br,base.state_br_mg,Serranos
-city_br_5062,base.br,base.state_br_pb,Serraria
-city_br_5063,base.br,base.state_br_ba,Serrinha
-city_br_5064,base.br,base.state_br_rn,Serrinha
-city_br_5065,base.br,base.state_br_rn,Serrinha dos Pintos
-city_br_5066,base.br,base.state_br_pe,Serrita
-city_br_5067,base.br,base.state_br_mg,Serro
-city_br_5068,base.br,base.state_br_ba,Serrolândia
-city_br_5069,base.br,base.state_br_pr,Sertaneja
-city_br_5070,base.br,base.state_br_pe,Sertânia
-city_br_5071,base.br,base.state_br_pr,Sertanópolis
-city_br_5072,base.br,base.state_br_rs,Sertão
-city_br_5073,base.br,base.state_br_rs,Sertão Santana
-city_br_5074,base.br,base.state_br_pb,Sertãozinho
-city_br_5075,base.br,base.state_br_sp,Sete Barras
-city_br_5076,base.br,base.state_br_rs,Sete de Setembro
-city_br_5077,base.br,base.state_br_ms,Sete Quedas
-city_br_5078,base.br,base.state_br_mg,Setubinha
-city_br_5079,base.br,base.state_br_rs,Severiano de Almeida
-city_br_5080,base.br,base.state_br_rn,Severiano Melo
-city_br_5081,base.br,base.state_br_sp,Severínia
-city_br_5082,base.br,base.state_br_sc,Siderópolis
-city_br_
```

Comment:
There are a bunch of useless changes, but I kept them because it was like that in the translation files